### PR TITLE
Docker and CI for ML

### PR DESCRIPTION
**Pull request by SeanChao**

### CODE REVIEW

**Comment by Hyridex** [comment]
LGTM

**Review by Hyridex — APPROVED** [review]

File: docker/machine_learning/download/checksums.sha256 [hunk 1]
```diff
@@ -0,0 +1,4 @@
+fc52756a74d5f371422ff99a7268cb6abb14bfc4a4de3569382cf0bd75e55d36  /data/models/Stable-diffusion/dreamlike-photoreal-2.0.ckpt
```
Comment (DanielSwarup) [inline docker/machine_learning/download/checksums.sha256:1]: should these be committed?

Comment (DanielSwarup) [reply docker/machine_learning/download/checksums.sha256:1]: Or should GitHub CI create these

File: docker/machine_learning/stable_diffusion_webui/Dockerfile [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+
+COPY clone.sh /clone.sh
+
+RUN . /clone.sh taming-transformers https://github.com/CompVis/taming-transformers.git 24268930bf1dce879235a7fddd0b2355b84d7ea6 \
+  && rm -rf data assets **/*.ipynb
+
+RUN . /clone.sh stable-diffusion-stability-ai https://github.com/Stability-AI/stablediffusion.git 47b6b607fdd31875c9279cd2f4f16b92e4ea958e \
+  && rm -rf assets data/**/*.png data/**/*.jpg data/**/*.gif
+
+RUN . /clone.sh CodeFormer https://github.com/sczhou/CodeFormer.git c5b4593074ba6214284d6acd5f1719b6c5d739af \
```
Comment (DanielSwarup) [inline docker/machine_learning/stable_diffusion_webui/Dockerfile:11]: I feel like these are secrets that we shouldn't be committing. Please correct me if I'm wrong

**Review by DanielSwarup — CHANGES_REQUESTED** [review]
In the future, please try to break this into smaller PR chunks.

File: docker/machine_learning/stable_diffusion_webui/Dockerfile (hunk 2 above)
Comment (SeanChao) [reply docker/machine_learning/stable_diffusion_webui/Dockerfile:11]: > I feel like these are secrets that we shouldn't be committing. Please correct me if I'm wrong

These are the git commit hashes

Comment (DanielSwarup) [reply docker/machine_learning/stable_diffusion_webui/Dockerfile:11]: Got it thanks.

File: docker/machine_learning/download/checksums.sha256 (hunk 1 above)
Comment (SeanChao) [reply docker/machine_learning/download/checksums.sha256:1]: These are the checksums for validating the model downloading result. They should be kept.

Comment (DanielSwarup) [reply docker/machine_learning/download/checksums.sha256:1]: Got it, thanks for the clarification

**Comment by DanielSwarup** [comment]
LGTM! The next steps would be to use a Docker Image of Blender so that we don't need to download it from the website each time.

**Review by DanielSwarup — APPROVED** [review]

**Pull request merged**